resources/definitions/fdmprinter.def.json

```diff
@@ -5686,7 +5686,7 @@
                     "minimum_value": "0",
                     "minimum_value_warning": "raft_surface_line_width",
                     "maximum_value_warning": "raft_surface_line_width * 3",
-                    "enabled": "resolveOrValue('adhesion_type') == 'raft'",
+                    "enabled": "resolveOrValue('adhesion_type') == 'raft' and raft_surface_layers > 0",
```

Comment:
`"enabled": "resolveOrValue('adhesion_type') == 'raft' and raft_surface_layers > 0"`

should be added to the following settings:

- "raft_surface_speed"
- "raft_surface_acceleration"
- "raft_surface_jerk"
- "raft_surface_fan_speed"
- "raft_surface_extruder_nr"

Reply:
We can't add it to the `raft_surface_extruder_nr` since that determines which extruder to get the setting from. But the rest, yes.